cosmos/coss.json

```diff
@@ -0,0 +1,79 @@
+{
+  "rpc": "https://coss-rpc.coss.ink",
+  "rest": "https://coss-rest.coss.ink",
+  "chainId": "coss-1",
+  "chainName": "Coss",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/coss/chain.png",
+  "nodeProvider": {
+    "name": "Coss Node",
+    "email": "coss.inscription@gmail.com"
+  },
+  "stakeCurrency": {
+    "coinDenom": "CGAS",
+    "coinMinimalDenom": "ucgas",
+    "coinDecimals": 6
+  },
+  "walletUrl": "https://coss.ink",
```

Comment:
@harrydevgo Could you remove `walletUrl` field?

Reply:
@HeesungB I have removed the `walletUrl` field.